src/bitmessageqt/__init__.py

```diff
@@ -2435,12 +2436,14 @@ def click_pushButtonAddAddressBook(self, dialog=None):
             ))
             return
 
-        self.addEntryToAddressBook(address, label)
-
-    def addEntryToAddressBook(self, address, label):
-        if shared.isAddressInMyAddressBook(address):
+        if address in BMConfigParser().addresses():
```

Comment:
This needs to be moved into the helper function. The error message can be done through UISignaller.